runtime/ftplugin/help.lua

```diff
@@ -66,51 +66,85 @@ vim.keymap.set('n', '[[', function()
   require('vim.treesitter._headings').jump({ count = -1 })
 end, { buffer = 0, silent = false, desc = 'Jump to previous section' })
 
--- Add "runnables" for Lua/Vimscript code examples.
----@type table<integer, { lang: string, code: string }>
-local code_blocks = {}
 local parser = assert(vim.treesitter.get_parser(0, 'vimdoc', { error = false }))
```

Comment:
I've kept the `parser` and `root` definition outside the `do...end` blocks so the code for "runnables" and for the gx implementation share those. Can also be repeated if that's cleaner.